torch/csrc/Module.cpp

```diff
@@ -1599,6 +1616,7 @@ static std::initializer_list<PyMethodDef> TorchMethods = {
      METH_NOARGS,
      nullptr},
     {"_to_dlpack", THPModule_toDLPack, METH_O, nullptr},
+    {"_to_dlpack_unversioned", THPModule_toDLPackUnversioned, METH_O, nullptr},
```

Comment:
Why do we still need this one?

Reply:
This does look necessary to me. For DLPack to change the ABI once, there's a dance that needs doing to be not-super-disruptive: continue returning the old (0.8) version, unless the consumer indicates it can handle the new (1.X) version by passing in `max_version=(1, 0)` (or `(1, x)` in the future).